hw/ip/ascon/data/ascon.hjson

```diff
@@ -0,0 +1,696 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+# Ascon register template
+{
+  name: "ascon",
+  regwidth: "32",
+  cip_id: "40",
+  version: "0.0.1",
+  clocking: [
```

Comment:
With https://github.com/lowRISC/opentitan/pull/22262 in place in parallel, I think you can already link the doc folder. The DV folder can also be linked, though it is not yet added.

Reply:
I can add them.

Reply:
At the moment they crash CI. I will add them later.